learning_assistant/api.py

```diff
@@ -224,3 +229,25 @@ def get_message_history(courserun_key, user, message_count):
     message_history = list(LearningAssistantMessage.objects.filter(
         course_id=courserun_key, user=user).order_by('-created')[:message_count])[::-1]
     return message_history
+
+
+def check_if_audit_trial_is_expired(user):
+    """
+    Given a user (User), get or create the corresponding LearningAssistantAuditTrial trial object.
+    """
+    audit_trial, created = LearningAssistantAuditTrial.objects.get_or_create(
+        user=user,
+        defaults={
+            "start_date": datetime.now(),
+        },
+    )
+
+    # If the trial was just created, then it definitely isn't expired, so return False
+    if created:
+        return False
+
+    # If the user's trial is expired, return True. Else, return False
```

Comment:
Related to above, but we may need to include logic for checking if the course upgrade deadline has passed (unless that's somewhere else, in which case, disregard!)